packages/docmanager-extension/src/index.ts

```diff
@@ -913,7 +924,7 @@ namespace Private {
     const confirmMessage = document.createElement('p');
     const confirmText = document.createTextNode(
       trans.__(
-        'Are you sure you want to revert the %1 to the latest checkpoint? ',
+        'Are you sure you want to revert the %1 to checkpoint? ',
```

Comment:
Maybe "to the selected checkpoint"?

Reply:
Even if there were no selection? (case where only one checkpoint exists)